src/toil/provisioners/gceProvisioner.py

```diff
@@ -686,6 +691,7 @@ def ex_create_multiple_nodes(
         """
          Monkey patch to gce.py in libcloud to allow disk and images to be specified.
          Also changed name to a uuid below.
+         The prefix 'wp' identifies preemptible nodes and 'wn' non-preemtible nodes.
```

Comment:
This is helpful! I'm curious what the w stands for.

Reply:
worker